dev-docs/publisher-api-reference.md

```diff
@@ -1215,4 +1216,74 @@ unsubscribe(); // no longer listening
 
+<a name="module_pbjs.dfp.buildVideoUrl"></a>
+
+### pbjs.adServers.dfp.buildVideoUrl(options) ⇒ `String`
+
+{: .alert.alert-info :}
+This method was added in 0.26.0.  For a usage example and instructions showing how to build Prebid.js to include this method, see [Show Video Ads with DFP]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+
+This method builds a DFP video ad tag URL by merging all of the bid data and publisher-supplied options into a single URL, and then returning it.
+
+It takes one argument, an `options` object (described [below](#adservers-dfp-options)) which should be used to construct the URL.
+
+It returns a (string) URL which calls DFP, letting `options.bid` (or the auction's winning bid for this adUnit, if undefined) compete alongside the rest of the demand in DFP.
+
+For more information about the options supported by the DFP API, see [the DFP API docs](https://support.google.com/dfp_premium/answer/1068325?hl=en#env).
+
+For more information about the arguments to this method, and example usage, see the sections below:
+
++ [Options](#adservers-dfp-options)
++ [Params](#adservers-dfp-params)
++ [Example](#adservers-dfp-example)
+
+<a name="adservers-dfp-options" />
+
+#### Options
+
+This object has the following properties:
+
++ `adUnit` (Object): The adUnit which this bid is supposed to help fill.  Required.
++ `bid` (Object): The bid which should be considered alongside the rest of the adserver's demand. If this isn't defined, then we'll use the winning bid for the adUnit. Optional.
++ `params` (Object): Query params which should be set on the DFP request. These will override this module's defaults whenever they conflict.  For more information, see the next section, [Params](#adservers-dfp-params).
+
+<a name="adservers-dfp-params" />
+
+#### Params
+
+This object contains the params needed to form a URL which hits the [DFP API](https://support.google.com/dfp_premium/answer/1068325?hl=en).
+
+All params (except `iu`, mentioned below) should be considered optional. This module will choose reasonable
+defaults for all of the other required params.
+
+The `params` object has the following properties:
+
++ `iu` (String): This param *must* be included, in order for us to create a valid request.
++ `description_url` (String): This field is required if you want Ad Exchange to bid on our ad unit... but otherwise optional.
++ `cust_params` (Object): If present, this will be merged with the rest of the standard Prebid targeting params (`hb_adid`, `hb_bidder`, etc).  For more information, see the [DFP documentation on `cust_params`](https://support.google.com/dfp_premium/answer/1068325?hl=en#cust_params) and the [Example](#adservers-dfp-example) below.
+
+<a name="adservers-dfp-example" />
+
+#### Example
+
+This example shows how to pass the `options` and `params` objects to the method.  For a fuller usage example, see [Show Video Ads with DFP]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
```

Comment:
Let's clarify this a bit.  Something like "This example calls pbjs.adServers.dfp.buildVideoUrl passing an options object with the adUnit and params fields.  The key-values present in the params.cust_params field will be merged with standard Prebid keys...".